git-filter-branch.sh

```diff
@@ -83,15 +83,21 @@ set_ident () {
 	finish_ident COMMITTER
 }
 
-if test -z "$FILTER_BRANCH_SQUELCH_WARNING$GIT_TEST_DISALLOW_ABBREVIATED_OPTIONS"
+if test -z "$FILTER_BRANCH_SQUELCH_WARNING$GIT_TEST_DISALLOW_ABBREVIATED_OPTIONS" && \
+   test "$(git config --bool filter-branch.squelchWarning)" != true
 then
 	cat <<EOF
 WARNING: git-filter-branch has a glut of gotchas generating mangled history
-	 rewrites.  Hit Ctrl-C before proceeding to abort, then use an
+	 rewrites. Hit Ctrl-C before proceeding to abort, then use an
```

Comment:
This is an independent change and needs to go to its own commit (together with an analogous change you made below).

You might find, though, that it is a contentious issue whether to use two or one space after a full stop. Personally, I find that two spaces are an anachronism when we had to worry about typewriters getting stuck on the rather small arm with the period character on it.

Having said that, if I were you, I'd save my energy for other causes and stick with the current state (i.e. two spaces).

Reply:
Reverted in latest commit bef1de4.